packages/vue-client/src/components/GameCreation/BadukConfigForm.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import type { BadukConfig } from "@ogfcommunity/variants-shared";
+
+const props = defineProps<{ initialConfig: BadukConfig }>();
+const config = { ...props.initialConfig } as BadukConfig;
+
+const emit = defineEmits<{
+  (e: "configChanged", config: BadukConfig): void;
+}>();
+
+function emitConfigChange() {
+  emit("configChanged", config);
+}
+</script>
+
+<template>
+  <div class="column">
```

Comment:
Maybe this `<div>` could be a `<form>` with a `@change`, then it wouldn't be necessary to add `@change` to each `<input>`.

Reply:
done